submission/loglist_refresher.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2019 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package submission
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"reflect"
+	"sync"
+	"time"
+
+	"github.com/google/certificate-transparency-go/loglist"
+	"github.com/google/certificate-transparency-go/x509util"
+)
+
+const (
+	// LoglistRefreshInterval is interval between consecutive external reads of Log-list.
+	loglistRefreshInterval = time.Hour * 24 * 7
+)
+
+// LogListEvent wraps result of single Log list refresh. Only one field
+// expected to be set.
+type LogListEvent struct {
```

Comment:
nit: You capitalise the "L" in "List" here, but don't capitalise it in other names (e.g. LoglistRefresher).

Reply:
Renamed.